components/create_comment/create_comment.jsx

```diff
@@ -980,9 +956,9 @@ class CreateComment extends React.PureComponent {
                         aria-label={emojiButtonAriaLabel}
                         type='button'
                         onClick={this.toggleEmojiPicker}
-                        className='style--none emoji-picker__container post-action'
+                        className={`emoji-picker__container post-action ${this.state.showEmojiPicker ? 'post-action--active' : ''}`}
```

Comment:
Consider using ```classnames``` for conditional classes. We want to introduce this practice everywhere.